tests/features/demo/demo_june.feature

```diff
@@ -0,0 +1,202 @@
+      | Who is Winter soldier?     | Captain America's child friend is alive?  | As it turns out the Hydra's agent-Winter soldier-is no other than Bucky-Captain's America childhood friend. | Project 'Captain America' |
+    And custom_page content:
+      | title              | body                                                                                                                              |
+      | S.H.I.E.L.D. Home  | Welcome to S.H.I.E.L.D. webspace. <br />You can find anything about S.H.I.E.L.D. here.                                            |
+      | About S.H.I.E.L.D. | S.H.I.E.L.D. stands for Strategic Homeland Intervention Enforcement and Logistics Division. That's all you need to know about it. |
+      | List of members    | Here is a list of members known to the public: <br><ul><li>Nick Fury</li></ul>                                                    |
+    And the following "custom_page" content belong to the corresponding collections:
+      | content            | collection   |
+      | S.H.I.E.L.D. Home  | S.H.I.E.L.D. |
+      | About S.H.I.E.L.D. | S.H.I.E.L.D. |
+      | List of members    | S.H.I.E.L.D. |
+    And the following "news" content belong to the corresponding collections:
+      | content                     | collection   |
+      | Phil Coulson is down        | S.H.I.E.L.D. |
+      | Phoenix is down             | x-Men        |
+      | S.H.I.E.L.D. is infiltrated | S.H.I.E.L.D. |
+    And the following "custom_page" content menu items for the corresponding collections:
+      | collection   | label              | page               | weight |
+      | S.H.I.E.L.D. | S.H.I.E.L.D. Home  | S.H.I.E.L.D. Home  | 1      |
+      | S.H.I.E.L.D. | About S.H.I.E.L.D. | About S.H.I.E.L.D. | 2      |
+      | S.H.I.E.L.D. | List of members    | List of members    | 3      |
+
+    # Scenario A. A collection owner manages his own collection.
+    And I am on the homepage
+    When I go to "/user"
+    And I fill in "Username" with "Nick Fury"
+    And I fill in "Password" with "ihaveasecret"
+    And I press "Log in"
+    # Login was successful if I see my profile page.
+    Then I should see the heading "Nick Fury"
+    And I should see the link "Collections"
+
+    # Collections overview.
+    When I click "Collections"
+    Then I should see the text "S.H.I.E.L.D."
+    And I should see the text "x-Men"
+    And I should see the text "Avengers"
+    And I should see the text "Fantastic four"
+
+    # Collection overview.
+    When I click "S.H.I.E.L.D"
+    # Solutions belonging to the collection.
+    Then I should see the text "Avengers initiative"
+    And I should see the text "Project Tahiti"
+    And I should see the text "Hellicarrier"
+    And I should see the text "Project 'Captain America'"
+    # News belonging to the solution.
+    And I should see the text "Phil Coulson is down"
+    And I should see the text "S.H.I.E.L.D. is infiltrated"
+    # News from solutions.
```

Comment:
Sorry, this is a typo. The first one is actually news belonging to the collection (not a solution) and the second one is news belonging to a solution. Will be fixed.